lesson_14/adrian_filimon/lesson_14.py

```diff
@@ -0,0 +1,80 @@
+"""
+Trucks drive around 500km a day and can work 5-6 days a week
+1km is payed for 0.85euro and everything after 2500km a week is payed with 1euro
+Make your class in such a way so that you can calculate the km payed and also the extra
+bonus for each km over 2500 km and the total income for 1 week.
+BONUS : if you know that 1L of Diesel costs 1,23E and a truck's consumption is 30L/100KM, how much does a
+payed km cost, with and without bonus?
+"""
+
+
+class PaymentMixin:
+
+    def __init__(self, *args):
+        class_name = type(self).__name__
+        print(f'{class_name} initialized.')
+
+        return super().__init__(*args)
```

Comment:
it is unusual for an __init__ method to return something. In this case, the super() is the object class. This line of code has no impact on the code overall you can remove it

Reply:
Will do 🤓
Thanks 😇